webflib.py

```diff
@@ -54,7 +70,9 @@ def login(self):
         except KeyError:
             http_proxy = None
         self.server = xmlrpclib.Server(API_URL, transport=http_proxy)
-        self.session_id, account = self.server.login(self.username, self.password)
+        extra_args = [self.machine] if self.machine else []
+        self.session_id, account = self.server.login(self.username, self.password, *extra_args)
+
         self.log.debug("self.session_id %s account %s" % (self.session_id,
```

Comment:
I know this is my messiness, but do you mind updating this to do the right thing and pass the parameters to the logging statement instead of interpolating right away? That way the interpolation only happens if the logger is set to `DEBUG`. So: `self.log.debug("self.session_id %s account %s", self.session_id, account)`